### DISCUSSION TITLE
Print line number and file name

### DISCUSSION THREAD

**Discussion by alessandro-bottamedi** [mowispace/react-native-logs#107] [Q&A]
I want to open this discussion to see if anyone has any ideas on how we can log the line number and file names... Feel free to share your ideas!

**Comment by tvillafane** [comment]
You would need to upload a sourcemap, [like Sentry does](https://docs.sentry.io/platforms/react-native/sourcemaps/).  I don't think there's a simple way to do this in a one-step fashion.
